runner/file.go

```diff
@@ -25,17 +26,33 @@ import (
 
 var preferredParallelism = runtime.NumCPU() * 2
 
-func transformCSV(in io.Reader, out io.Writer, delimiter rune) error {
+const ParallelEncodingMin = 100 * 1024 * 1024 // 100 MB
+
+func transformCSV(in io.Reader, out io.Writer, delimiter rune, parallelEncoding bool) error {
```

Comment:
Since CSV processing is getting larger now let's pull this out into a `file_csv.go` dedicated file for CSV stuff similar to the various database_X.go files